### ISSUE TITLE
Loggly Transport stops working with winston after a while

### ISSUE THREAD

**Issue by satgrocks** [loggly/winston-loggly-bulk#73]
Hello, So we are using the winston lib ver 3.3.3 along with winston-loggly-bulk 3.1.1 and we are able to see the logs in Console (using ConsoleTransport of winsto) and loggly dashboard as well. But after a few seconds, the logs are not sent to loggly anymore, but the console logs are still available.  I checked the transports attributes of the logger (winston.Logger), I could see both the ConsoleTransport and Loggly (winston-loggly-bulk) in there. 
If I switch to old versions winston lib ver 3.1.0 along with winston-loggly-bulk 2.0.3, I see the logs go to loggly all the time. Appreciate any help on this.

```
        const CustomLogLevels = {
            levels: levelMapping
        };
        this.logger = winston.createLogger({
            level: 'verbose',
            levels: CustomLogLevels.levels,
            exitOnError: false,
            format: winston.format.simple()
        });

let options = {
            level: 'verbose',
            token: inputToken,
            subdomain: subdomain,
            tags: [moduleName],
            json: true,
            isBulk: true,
            stripColors: true
        };
        this.logglyTransport = new Loggly(options);
        this.logger.add(this.logglyTransport);

```

**Comment by alestrunda** [comment]
Hello, we're having some issues with `isBulk: true` option. When I was testing it I was experiencing the exact same problem you're describing. We're currently working on that, but for now, I can only recommend not to use `isBulk` option.

**Comment by satgrocks** [comment]
@alestrunda Thanks for your response. We had issues while sending logs to loggly from front end (UI) without the isBulk option for some reason. For now I guess we will just wait till you have a solution for this problem. Thanks again.

**Comment by satgrocks** [comment]
@alestrunda Hi there. Are there any updates on this issue ?

**Comment by alestrunda** [comment]
Sorry about the delay, there were other priorities, anyway the root cause of this issue is in lib `node-loggly-bulk` - the fix is ready and just being reviewed https://github.com/loggly/node-loggly-bulk/pull/62

**Comment by satgrocks** [comment]
Thanks for the update. Please let know once it is merged.

**Comment by michal-bures** [comment]
Fixed by #75 (release `v3.2.1`). @satgrocks Feel free to reopen in case the issue persists.

**Comment by alestrunda** [comment]
@satgrocks fix has been released, but that's just the lib https://github.com/loggly/node-loggly-bulk so we will also have to update that dependency for winston-loggly-bulk and release it. Not sure when that's going to happen, we're currently doing maintenance of several open-source libs we've got.

Anyway you can get the fix when you go to `node_modules/winston-loggly-bulk` and run `npm install node-loggly-bulk@3.0.0`, I checked it, seem to be working fine.
